org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/WListbox.java

```diff
@@ -1255,5 +1277,100 @@ public void setAllowIDColumnForReadWrite(boolean allowIDColumnForReadWrite)
 	{
 		this.allowIDColumnForReadWrite = allowIDColumnForReadWrite;
 	}
+	
+	   /**
+     * Retrieve the width of the column
+     * 
+     * @param columnName the name of the column to find the width of.
+     * @param columnList list of the columns and their width
+     * @return the width of the column
+     */
+	private String getColumnWidth(String columnName, List<String>columnList) 
+	{
+		String width ="null";
+		String[] w;
+		
+		for (String column : columnList)
+		{
+			w = column.trim().split("=");
+			if (w[0].equals(columnName))
+			{
+				return w[1];
+			}
+		}
+		return width;
+	}
+	
+    /**
+     * Render the header of the WListbox with the columns width of the saved column width
+     * 
+     */
+	public void renderHeaderColumnWidth () 
```

Comment:
this perhaps doesn't needs to be public and it is more obvious to name this as renderCustomHeaderWidth() instead.

Reply:
keeping it as public, but changing the name to renderCustomHeaderWidth as it makes more sense.